### ISSUE TITLE
[A11y]"Learn more" link is not descriptive.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7879]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1062807
Steps to Reproduce:
1. Hit the #URL:https://www.nuget.org/
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Select “Manage Organizations” from the Profile Dropdown and hit enter.
5. Navigate to "Add New" control and hit enter.
6. Turn on narrator, navigate to "Learn" more link using Tab key.
7. Observe Narrator announcement.

Actual:
Narrator not announces "Learn More" link in descriptive. 
When "Learn More" link receives Narrator focus it does not describe what the learn more link means.

Same issue is Observed in "Chrome+NVDA" also.

Same issue is Observed for controls "Learn More" and "Read More" in  Scenario "UploadPackage_Verify". Below is the screenshot.
Image
Expected:
The narrator should announce a short description about the learn more link.

Notes:
Not exactly sure what is expected to be announced here since "Learn more" is already in the context of the organizations page. We could update the "Learn More" link to include an aria-label with the text "Learn more about organizations on NuGet.Org"
